rasa/core/featurizers.py

```diff
@@ -578,19 +515,64 @@ def slice_state_history(
 
         slice_end = len(states)
         slice_start = max(0, slice_end - slice_length)
```

Comment:
let's add handling `slice_length=None` as infinite max history

Reply:
should I get rid of `FullDialogueTrackerFeaturizer`? and change the standard featurizer in all the policies?